src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -124,6 +124,7 @@ export default class SourceFileLinter {
 				node as (ts.PropertyAccessExpression | ts.ElementAccessExpression)); // Check for global
 			this.analyzePropertyAccessExpressionForDeprecation(
 				node as (ts.PropertyAccessExpression | ts.ElementAccessExpression)); // Check for deprecation
+			this.analyzeExportedValuesByLib(node as ts.PropertyAccessExpression);
```

Comment:
`node` could also be a `ElementAccessExpression`

Reply:
Thanks!